dev-python/sphinxcontrib-bibtex/sphinxcontrib-bibtex-2.1.4.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Sphinx extensions for BibTeX style citations"
+HOMEPAGE="https://github.com/mcmtroffaes/sphinxcontrib-bibtex"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/docutils[${PYTHON_USEDEP}]
+	dev-python/pybtex[${PYTHON_USEDEP}]
+	dev-python/pybtex-docutils[${PYTHON_USEDEP}]
+	dev-python/sphinx[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests pytest
```

Comment:
> Fixed with `dev-python/PyQt5[testlib]`

Hmm, this is a bit strange, I don't see anything in the source code that calls PyQt5/PySide2 at all, just some references to qthelp.

In any case, I dropped the sphinxcontrib-bibtex commit from this PR because @mgorny has already pushed version 2.2.0 to ::gentoo